src/sentry_plugins/github/endpoints/webhook.py

```diff
@@ -99,7 +99,10 @@ def __call__(self, organization, event):
                                 author_email = user.email
                                 gh_username_cache[gh_username] = author_email
                                 if commit_author is not None:
-                                    commit_author.update(email=author_email)
+                                    commit_author.update(
+                                        email=author_email,
+                                        external_id=gh_username,
```

Comment:
why is that a big deal? we'd think you were the same author, which you are. i think the case where it would be bad is where you're something in github and then i have the same username that you do in github in phabricator.